src/@carbon-info/views/Ecosystem/Ecosystem.tsx

```diff
@@ -64,18 +65,100 @@ const Ecosystem: React.FC = () => {
+  const dAppsFilterButtons = (
     <Box className={classes.filters}>
-      {filters.map((filter, index) => {
+      {dAppsFilters.map((filter, index) => {
+        return (
+          <Box className={classes.filterBox} key={`${filter}-${index}`}>
+            <Button className={clsx(classes.filterButton, { active: dAppsFilter === filter })} onClick={() => setDAppsFilter(filter)} disableRipple>
+              {filter}
+            </Button>
+          </Box>
+        );
+      })}
+    </Box>
+  );
+
+
+
+  const filteredDApps = React.useMemo(() => {
+    let filtered: DAppsConfig[] = [];
+    switch (dAppsFilter) {
+      case "Carbon Core":
+        filtered = allDApps.filter((blockchain) => {
+          return blockchain.category === "Carbon Core";
+        });
+        break;
+      case "Carbon EVM":
+        filtered = allDApps.filter((blockchain) => {
+          return blockchain.category === "Carbon EVM";
+        });
+        break;
+      case "All":
+        filtered = allDApps;
+        break;
+      default:
+        filtered = allDApps;
+        break;
+    }
+    return filtered;
+  }, [allDApps, dAppsFilter]);
+
+  const renderSubtext = React.useMemo(() => {
+    let subtext;
+    switch (dAppsFilter) {
+      case "Carbon EVM":
+        subtext = (
+          <>
+            <Typography variant="body1" color="textSecondary" align="left" style={{ maxWidth: "1024px", marginTop: "20px" }}>
+              The EVM component of Carbon allows anyone to deploy EVM-based smart contracts written in Solidity, Vyper, etc. on Carbon. This component allows users to perform any action via both Cosmos and EVM formatted transactions, meaning that users and developers can use popular Ethereum wallets and clients (e.g. MetaMask, HardHat, etc.) to interact with Carbon without additional effort.
+            </Typography>
+            <CTAButton
+              text="Learn More"
+              link={Path.Docs.CarbonEVM}
+              textClassName={classes.ctaButtonText}
+              iconClassName={classes.ctaButtonIcon}
+            />
+          </>
+        );
+        break;
+      case "Carbon Core":
+        subtext = (
+          <>
+            <Typography variant="body1" color="textSecondary" align="left" style={{ maxWidth: "1024px", marginTop: "20px" }}>
+              Carbon Core consists of various native modules written in native code (e.g. Golang) instead of a virtual machine. This implementation securely enables features in a scalable manner such as on-chain central-limit order books, lending and borrowing markets, and more.
+            </Typography>
+            <CTAButton
+              text="Learn More"
+              link={Path.Docs.CarbonCore}
+              textClassName={classes.ctaButtonText}
+              iconClassName={classes.ctaButtonIcon}
+            />
+          </>
+        );
+        break;
+      default:
+        subtext = null;
+    }
+    return subtext;
+  }, [allDApps, dAppsFilter]);
```

Comment:
1. Think we should try not to `useMemo` HTML markup code. useMemo is usually more applicable to data (e.g. strings/numbers/objects). Instead, it would be better to apply `useMemo` for the description and the cta link, then conditionally render the HTML markup code directly in the return statement.
i.e.
```typescript
interface SubTextContent {
  description: string
  link: string
}

const subTextContent = React.useMemo((): SubTextContent | null => {
  switch (dAppsFilter) {
    case "Carbon EVM":
      return {
        description: 'The EVM component of Carbon allows...',
        link: Paths.Docs.CarbonEVM,
      }
    case "Carbon Core":
      // return statement here
    default:
      return null
  }
}, [dAppsFilter])
...
<>
  ...
  {subTextContent && (
    <>
      <Typography variant="body1" color="textSecondary" align="left" style={{ maxWidth: "1024px", marginTop: "20px" }}>
        {subTextContent.description}
      </Typography>
      ...
    </>
  )}
</>
```

2. Also, try not to do excessive styling in the HTML style property. If there is more than 1 style variable, should move the styles to a CSS class instead.